test/sys/test_socket.rs

```diff
@@ -1428,13 +1428,13 @@ pub fn test_vsock() {
                     SockFlag::empty(), None)
              .expect("socket failed");
 
-    // VMADDR_CID_HYPERVISOR and VMADDR_CID_RESERVED are reserved, so we expect
+    // VMADDR_CID_HYPERVISOR and VMADDR_CID_LOCAL are reserved, so we expect
```

Comment:
This change looks unrelated.  Where did it come from?

Reply:
from build warning.

https://docs.rs/libc/0.2.79/libc/constant.VMADDR_CID_RESERVED.html 
```
👎 Deprecated since 0.2.74:
VMADDR_CID_RESERVED is removed since Linux v5.6 and replaced with VMADDR_CID_LOCAL 
```